doc_source/reference_aws-services-that-work-with-iam.md

```diff
@@ -279,4 +279,4 @@ The AWS services listed below are grouped by their [AWS product categories](http
 | --- |--- |--- |--- |--- |--- |--- |
 |  Service  |  Actions  |  Resource\-level permissions  | Resource\-based policies |  Authorization based on tags  |  Temporary credentials  |  Service\-linked roles  | 
 |  [AWS Billing and Cost Management](http://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/ControllingAccessWebsite.html)  | Yes | No | No | No | Yes | No | 
-|  [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html#accessing-support)  | No | No | No | No | Yes | No | 
\ No newline at end of file
+|  [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html#accessing-support)  | No | No | No | No | Yes | No | 
```

Comment:
Argh, sorry about the spurious diff here. I just used GitHub's web editor. Will try to fix when I get somewhere I can use the command-line again.